src/bp/core/services/workspace-service.ts

```diff
@@ -57,16 +57,16 @@ export class WorkspaceService {
 
   listUsers(selectFields?: Array<keyof AuthUser>): Partial<AuthUser[]> {
     if (!selectFields) {
-      return this._workspace.users
+      return this._workspace && this._workspace.users
```

Comment:
is it even possible to have no selected workspace ?

Reply:
It was to prevent trying to read the users list before the workspace is loaded, but not mandatory